app/src/main/java/io/multy/ui/fragments/send/AssetSendFragment.java

```diff
@@ -115,7 +117,16 @@ public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2)
 
             @Override
             public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
-                if (TextUtils.isEmpty(charSequence)){
+                boolean isValidAddress = false;
```

Comment:
mark as //TODO to improve this method. It would be better to retur boolean values and replace these lines with boolean  isValidAddress = NativeDataHelper.isAddressValid(address); instead of try catch and code spaming.